src/main/java/org/victorrobotics/frc/dtlib/sensor/encoder/ctre/DTCANCoder.java

```diff
@@ -0,0 +1,126 @@
+package org.victorrobotics.frc.dtlib.sensor.encoder.ctre;
+
+import org.victorrobotics.frc.dtlib.sensor.encoder.DTAbsoluteEncoder;
+import org.victorrobotics.frc.dtlib.sensor.encoder.DTAbsoluteEncoderFaults;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.simulation.EncoderSim;
+
+import com.ctre.phoenix.sensors.CANCoder;
+import com.ctre.phoenix.sensors.CANCoderFaults;
+
+public class DTCANCoder implements DTAbsoluteEncoder {
+    private final CANCoder internal;
+
+    private String         firmwareVersion;
+
+    public DTCANCoder(int canID) {
+        internal = new CANCoder(canID);
+
+        internal.setPositionToAbsolute();
+    }
+
+    public DTCANCoder(int canID, String canBus) {
+        internal = new CANCoder(canID, canBus);
```

Comment:
Only one of the constructors calls `setPositionToAbsolute`, is that intentional?

Reply:
Also I recall that the cancoder can be configured to return values from -180 to 180, or 0 to 360, and it seems like we might want different zero positions depending on the application.

Reply:
Fixed the constructor, and that is a good addition